packages/lib/response/service.ts

```diff
@@ -80,86 +83,98 @@ const responseSelection = {
   },
 };
 
-export const getResponsesCacheTag = (surveyId: string) => `surveys-${surveyId}-responses`;
-
-export const getResponseCacheTag = (responseId: string) => `responses-${responseId}`;
+const getResponseBySingleUseIdCacheTags = (surveyId: string, singleUseId: string): string =>
```

Comment:
I think this should also move to `cache.ts`?

Reply:
will remove, and also all the functions in `cache.ts`